### PR TITLE
Check repository health, setting up continuous integration and proposed frontend of project

### PR DESCRIPTION
**Pull request by eshaibu**
#### What does this PR do?

* Checking repository health 
* Setting up continuous integration
* Front-end template of proposed inverted index project

#### Description of Task to be completed?

* Checks repository health using `Code Climate` and `Coveralls`
* Setting up continuous integration using TravisCI to show develpment build
* A folder containing the  proposed front-end template of inverted index project using HTML, CSS and JavaScript

#### How should this be manually tested?

**_To see the repository health click the link below to see  GPS badges and code coverage in the README.md_** 
* https://github.com/andela-eshaibu/inverted-index/

**_To test setting up continuous integration using TravisCI_**
* https://travis-ci.org/andela-eshaibu/inverted-index/builds/202686894

**_To test and see the proposed project front end template_**
Click on this link https://github.com/andela-eshaibu/inverted-index/tree/feature/139952859/frontend-template/UI to view the folder containing the HTML, CSS and JS of proposed project template
  - To see visual of it type the following command in your terminal
```
git clone https://github.com/andela-eshaibu/inverted-index.git
cd inverted-index
git checkout feature/139952859/frontend-template
npm install
gulp
```
#### Any background context you want to provide?

N/A
#### What are the relevant pivotal tracker stories?

* Checking repository health
* Setting up continuous integration
* Front-end template of proposed inverted index project

#### Screenshots (if appropriate)

**_Screenshot showing code coverage and GPA badges in README file_** 
 badges

**_Screenshot showing front end template_**  
frontend

#### Questions:

N/A

### CODE REVIEW

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#### Expected Output
+- **Using**: HTML, CSS & JS
+- **Deliver**: a folder containing the template for the proposed frontend of your project
+- **With**: *independently*
+
+#### Output
+
+- [Link to folder containing template for proposed frontend of project](https://github.com/andela-eshaibu/inverted-index/tree/feature/139952859/frontend-template/UI)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/README.md]: Kindly provide an image of the UI so I can see what it looks like.

File: 06-building-a-program-start-to-finish/setting-up-continuous-integration-(build-server)/output/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+- **Using**: TravisCI
+- **Deliver**: a link to a successful TravisCI build of your `development` branch (it should include test runs)
+- **With**: *independently*
+
+#### Link to github repo
+- [andela-eshaibu/inverted-index](https://github.com/andela-eshaibu/inverted-index)
+
+#### Output
+
+- [Link to succesful travis build of development branch](https://travis-ci.org/andela-eshaibu/inverted-index/builds/202686894)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-continuous-integration-(build-server)/output/README.md:11]: Great Job!

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/README.md (hunk 1 above)
Comment (eshaibu) [reply 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/README.md]: @q-ode I have included the image of the UI for you to see

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+#### Screenshots
+
+**_Screenshot of proposed frontend template_**
+ ![frontend](https://cloud.githubusercontent.com/assets/25608317/23165226/5735e49a-f83b-11e6-8b6e-084e903b2247.JPG)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/README.md:13]: Great Job!

You might want to rethink the idea of having multiple `Create Index` button elements on the left. What happened when 50 files are uploaded?

I'll accept this but expect that you fix this.

**Pull request merged**